hypha/apply/funds/tables.py

```diff
@@ -257,13 +258,18 @@ class SubmissionFilter(filters.FilterSet):
         (50, '50'),
         (100, '100'),
     )
+
     round = Select2ModelMultipleChoiceFilter(queryset=get_used_rounds, label='Rounds')
     fund = Select2ModelMultipleChoiceFilter(field_name='page', queryset=get_used_funds, label='Funds')
     lead = Select2ModelMultipleChoiceFilter(queryset=get_round_leads, label='Leads')
     reviewers = Select2ModelMultipleChoiceFilter(queryset=get_reviewers, label='Reviewers')
     screening_statuses = Select2ModelMultipleChoiceFilter(queryset=get_screening_statuses, label='Screening', null_label='No Status')
+    category_options = Select2MultipleChoiceFilter(
+        choices=[], label='Category Options',
```

Comment:
Make label only "Category", the filters takes to mush space as it is.

Reply:
Renamed it.